src/controllers/teamCtrl.js

```diff
@@ -216,10 +263,24 @@ exports.getUserOfTeam = (req, res) => {
   Team.findById(req.params.teamId)
     .populate(['users.user'])
     .then((team) => {
+      if (team) {
```

Comment:
```suggestion
      if (!team) {
```
Its better for a not found 😂

Reply:
> ```suggestion
> 
>       if (!team) {
> 
> ```
> 
> Its better for a not found 😂

Good catch! Thx!